LoginPage.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+
+const LoginPage = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await axios.post('/api/login', { email, password });
+      history.push('/home');
+    } catch (err) {
+      console.error(err);
+    }
+  };
```

Comment:
Avoid using anonymous functions inside the render method. This can lead to unnecessary re-renders as a new function is created every time the component re-renders.